app/controllers/tag_controller.rb

```diff
@@ -132,8 +132,14 @@ def show
     end
     nodes = nodes.where(created: @start.to_i..@end.to_i) if @start && @end
 
-    @notes = nodes.where('node.nid NOT IN (?)', qids) if @node_type == 'note'
-    @questions = nodes.where('node.nid IN (?)', qids) if @node_type == 'questions'
+    if qids.empty?
```

Comment:
Hm, could we move this line so it's just above here, and a bit easier to read? https://github.com/publiclab/plots2/pull/5141/files#diff-a8eb2425ebeb0814d010864bb014e815R97

Reply:
good idea

Reply:
ah actually, that's not possible, because `@notes` and `@questions` depend on `nodes` so these have to be set after `nodes` is set. Instead I moved the creation of `qids` down here.